(node:3808641) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'banner-session-45578',
  containerIp: '10.0.3.11',
  mitmPort: 45578,
  autoAccessFixed: 1
}
2024-11-15 08:15:58.550 - [Info] MITM Version: 2
2024-11-15 08:15:58.552 - [Info] Auto Access Enabled: true
2024-11-15 08:15:58.552 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-15 08:15:58.562 - [Info] SSH man-in-the-middle server for 10.0.3.11 listening on 10.0.3.1:45578
2024-11-15 08:24:02.747 - [Debug] [Connection] Attacker connected: 128.199.73.168 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-15 08:24:04.236 - [Debug] [Auth] Attacker 128.199.73.168 trying to authenticate with 'password'
2024-11-15 08:24:04.237 - [Debug] [Auto Access] Attacker: 128.199.73.168, Threshold: 1, Attempts: 1
2024-11-15 08:24:04.237 - [Debug] [Auto Access] Compromising the honeypot
2024-11-15 08:24:04.237 - [Debug] [Auto Access] Adding the following credentials: 'natali:1234'
2024-11-15 08:24:04.485 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-15 08:24:04.487 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-15 08:24:04.488 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.11
2024-11-15 08:24:04.567 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-15 08:24:05.520 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-15 08:24:07.059 - [Debug] [Connection] Attacker connected: 128.199.73.168 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-15 08:24:08.628 - [Debug] [Auth] Attacker 128.199.73.168 trying to authenticate with 'password'
2024-11-15 08:24:08.629 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-15 08:24:08.630 - [Debug] [Auth] Attacker: 128.199.73.168 has so far made 1 attempts. Remaining: 5 attempts
2024-11-15 08:24:09.147 - [Debug] [Connection] Attacker closed the connection
2024-11-15 08:24:09.419 - [Debug] [Connection] Attacker connected: 128.199.73.168 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
